From f901c5c17b15ae979cca5a3da7247404ffc8995e Mon Sep 17 00:00:00 2001
From: Alexander Gordeev <lasaine@lvk.cs.msu.su>
Date: Wed, 6 Oct 2010 14:53:22 +0400
Subject: [PATCH 05/20] remove duplicate typedef for u8

This typedef is already in eap_peer/util/common.h, no need to declare
it one more time.

Signed-off-by: Alexander Gordeev <lasaine@lvk.cs.msu.su>
---
 .../WiMax/Wrappers/Supplicant/wmxSDK_Sup_Impl.c    |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/InfraStack/OSAgnostic/WiMax/Wrappers/Supplicant/wmxSDK_Sup_Impl.c b/InfraStack/OSAgnostic/WiMax/Wrappers/Supplicant/wmxSDK_Sup_Impl.c
index a33680c..8f55917 100644
--- a/InfraStack/OSAgnostic/WiMax/Wrappers/Supplicant/wmxSDK_Sup_Impl.c
+++ b/InfraStack/OSAgnostic/WiMax/Wrappers/Supplicant/wmxSDK_Sup_Impl.c
@@ -47,7 +47,6 @@
 #include "NDnSAgent.h"
 
 
-typedef unsigned char u8;
 #define TLS_IMPLEMENTATION Arm1
 
 /* the supplicant calls the tls_ methods, so we make sure the declarations match to what it expects. */ 
-- 
1.7.4.1

